### ISSUE TITLE
Multiple observations for a single time point

### ISSUE THREAD

**Issue by pshankinclarke** [SugiharaLab/pyEDM#7]
Is it possible to use multiple observed values for a single time point in a data sequence? Thank you!

**Comment by SoftwareLiteracy** [comment]
Yes, this is the function of the embedded = True option, indicating
    that the input is already an embedding, use it "as-is". 
    
    As noted in the docs:
    Parameters
      - EDM (pyEDM, rEDM)
    
    
      
        
          embedded
          Logical specifying if the input data are embedded. If embedded
              = True, no embedding is performed. If embedded
              = False, the input data are embedded to dimension E
            with time shift tau.
        
      
    
    
    If embedded = True, then the columns specified are used directly as
    the state-space embedding.  The user first constructs their data
    frame with columns containing the desired observations.  
    
    An example using columns "x_t y_t z_t" from :
    
    >>>
        pyEDM.sampleData["block_3sp"].head(3)
         time       x_t     x_t-1     x_t-2  ...    
        y_t-2       z_t     z_t-1     z_t-2
      0     3 -1.917685  1.244882 -0.741863  ... -1.268104 
        1.535239 -0.481583 -1.863980
      1     4 -0.962318 -1.917685  1.244882  ...  1.488887
        -1.492956  1.535239 -0.481583
      2     5  1.331875 -0.962318 -1.917685  ... -0.113188
        -1.119476 -1.492956  1.535239
      
      >>> s = pyEDM.Simplex( dataFrame =
        pyEDM.sampleData["block_3sp"], lib = "1 99", pred = "100 195", 
        E = 3, embedded = True, columns="x_t y_t z_t", target="x_t" )
      
      >>> s.tail()
         time  Observations  Predictions  Pred_Variance
      92  194      0.784424     0.722976       0.069653
      93  195     -1.277411    -0.705869       0.622008
      94  196      0.558733     0.711675       0.028580
      95  197      0.633977     0.372280       0.082721
      96  198           NaN     0.155835       0.122911
    
    
    

    On 8/10/20 9:13 PM, pshankinclarke
      wrote:
    
    
      
      Is it possible to use multiple observed values for a single
        time point in a data sequence? Thank you!
      —
        You are receiving this because you are subscribed to this
        thread.
        Reply to this email directly, view it on GitHub, or unsubscribe.
      [
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/SugiharaLab/pyEDM/issues/7",
"url": "https://github.com/SugiharaLab/pyEDM/issues/7",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
